eng/pipelines/coreclr/templates/run-performance-job.yml

```diff
@@ -69,28 +69,28 @@ jobs:
       - group: DotNet-HelixApi-Access
       - group: dotnet-benchview
 
-    - ${{ if not(and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'))) }}:
+    # public (PR) job settings
+    - ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
       - ${{ if eq( parameters.osGroup, 'Windows_NT') }}:
         - HelixPreCommand: $(ExtraMSBuildLogsWindows)
       - ${{ if ne(parameters.osGroup, 'Windows_NT') }}:
-          - HelixPreCommand: $(ExtraMSBuildLogsLinux);npm install --prefix $HELIX_WORKITEM_PAYLOAD jsvu -g;$HELIX_WORKITEM_PAYLOAD/bin/jsvu --os=linux64 --engines=v8
+        - HelixPreCommand: $(ExtraMSBuildLogsLinux);npm install --prefix $HELIX_WORKITEM_PAYLOAD jsvu -g;$HELIX_WORKITEM_PAYLOAD/bin/jsvu --os=linux64 --engines=v8
     
 
     - ${{ if and(eq(parameters.codeGenType, 'Interpreter'), eq(parameters.runtimeType, 'mono')) }}:
       - ${{ if eq( parameters.osGroup, 'Windows_NT') }}:
+        - Interpreter: ' -MonoInterpreter'
```

Comment:
This seems to change the logic of the statements. Before we only had this flag set if we were not in public. It not seems that we set this all the time. This is not the behavior that we want as we don't want to run interpreter runs except on internal queues.

Reply:
hmm I can see mono interpreter is set for both public and private jobs originally? https://github.com/dotnet/runtime/blob/60eff3f3766631bd4e7ee256ca17619fec90e9e6/eng/pipelines/coreclr/templates/run-performance-job.yml#L87-L93 In public jobs ``` --monointerpreter``` is specified https://dev.azure.com/dnceng/public/_build/results?buildId=775941&view=logs&j=fb000423-50a9-5897-1483-627612d50fb2&t=94a2964d-63c2-56af-48bf-b02f5cb84eaa&l=11

Reply:
OK, looking at what it is before I see what is going on. This was turned off for public, pr triggered runs, but not regular batched public runs. In general, I think that we should make sure that we are no longer running any public runs except when we are making a change and want to turn on PR runs for the performance legs for testing. But we can do this cleanup in another commit.

Reply:
Yeah, that makes sense.